test/test_dss_cli.py

```diff
@@ -192,13 +192,21 @@ def test_parsing(self):
         self.assertEqual(cli.parse_args(args), out)
 
         args = ["put-files", "--creator-uid", "1", "--source-url", "sljdf.com", "134"]
-        self.assertRaises(hca.dss.error.PrintingException, cli.parse_args, args)
+        process = subprocess.Popen(["hca"] + args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Does this actually test against the current repo?  I would imagine this would test against the installed binary.

Reply:
Indeed.  It tests against the currently installed hca script.  The first step in the Makefile for the "test" target is to install the package.  I didn't change this.  I assume there is some good reason for it cooked up by James/Andrey.   Several of the tests are written this way. 
 eg. https://github.com/HumanCellAtlas/data-store-cli/blob/567854d0d57d286a8aa0b34f84fc8035bde54a4a/test/test_dss_cli.py#L174

Reply:
Well, I guess the cost to install the package isn't that significant... Just potentially quite awkward for people not in a virtualenv.

Reply:
Lol yeah this was my trigger to start using virtualenv :/